src/Core/IProcessorArchitecture.cs

```diff
@@ -38,7 +38,7 @@ public interface IProcessorArchitecture
         /// Creates an IEnumerable of disassembled MachineInstructions which consumes 
         /// its input from the provided <paramref name="imageReader"/>.
         /// </summary>
-        /// <remarks>This was previously an IEnumerator, but making it IEnumerable lets us use Linq expressions
+        /// <remarks>This was previously an IEnumerator, but making it IEnumerable lets us use LINQ expressions
```

Comment:
@chostelet please do such minor doc changes in separate commit(s) in branch(es) starting from `master`